### DISCUSSION TITLE
Keeps loading after clicking request, but other menu options are available

### DISCUSSION THREAD

**Discussion by 819965653** [seerr-team/seerr#1159] [Q&A] [answered]
<img width="1182" alt="image" src="https://github.com/user-attachments/assets/b8174590-4955-43a4-aca1-61e14ab76381" />

**Comment by fallenbagel** [comment, marked as answer]
This is actually both a bug and a configuration issue.

Your issue is you have a connection issue on one of your *arrs in service. Just go to settings > services and press test on all your services.

The bug part is that it should not mess up if there is a connection issue and this has already been fixed in #1141
